website/screens/components/dialog/code/examples/basicUsage.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  DxcDialog,
+  DxcButton,
+  DxcInset,
+  DxcRow,
+} from "@dxc-technology/halstack-react";
+import { useState } from "react";
+
+const code = `() => {
+  const [isDialogVisible, setDialogVisible] = useState(false);
+  const handleClick = () => {
+    setDialogVisible(!isDialogVisible);
+  };
+  return (
+    <DxcInset space="large">
+      <DxcRow justify="center">
+        <DxcButton label="Open Dialog" onClick={handleClick}></DxcButton>
+        {isDialogVisible && (
+          <DxcDialog onCloseClick={handleClick}>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua.
+          </DxcDialog>
+        )}
+      </DxcRow>
+    </DxcInset>
```

Comment:
I would not use DxcRow in order to have it not centered, as in the rest of components.